src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './variables';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: $default-font;
+  font-size: 30px;
+}
+
+.calendar {
+  width: $calendar-width;
+  padding: $calendar-padding;
+
+  display: flex;
+  flex-flow: row wrap;
+  gap: $cell-gap;
+}
+
+.calendar__day {
+  height: $cell-size;
+  width: $cell-size;
+  border: $cell-border-size solid black;
+  background-color: #eee;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition-property: transform, background-color;
+  transition-duration: 0.5s;
```

Comment:
It's a good practice to specify `transition-timing-function` property to control the speed curve of the transition. It can make the transition look more smooth.